lib/topic.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, Page } from '@playwright/test';
+import { waitForKafkaReady } from './kafka';
+
+export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
+  await expect(page.getByText(kafkaName)).toHaveCount(1);
+  await waitForKafkaReady(page, kafkaName);
+  await page.getByText(kafkaName).click();
+  await page.getByText('Topics').click();
+};
+
+export const createKafkaTopic = async function (page: Page, name: string, check = true) {
+  await page.getByText('Create topic').click();
+  await expect(page.getByText('Create topic')).toHaveCount(2);
+  // This is default Topic creation
+  await page.getByPlaceholder('Enter topic name').fill(name);
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Finish').click();
+
+  if (check) {
+    // check that the instance has been created
+    const table = page.locator('[data-ouia-component-id=card-table]');
+    expect(table.getByText(name)).toBeTruthy();
```

Comment:
`toHaveCount(1)` maybe?

Reply:
This indeed does make sense. Though the chances are low, `toHaveCount(1)` would test also that there is no other instance with the same name - which should be almost impossible. 

Furthermore, If I'm reading the [documentation](https://playwright.dev/docs/api/class-locator#locator-get-by-text-option-exact) of `getByText` correctly, there is a chance it could give you also an instance that has `name` as a substring of its name. 
I know that with the current design of the test suite there should not be any other instances present in the org, but if something like cleanup fails...
`expect(table.getByText(name, { exact: true })).toHaveCount(1);` Would make this even more resilient to false positives because, by default, exact matching is set to false.